pkg/virtctl/install/install.go

```diff
@@ -0,0 +1,56 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2017, 2018 Red Hat, Inc.
+ *
```

Comment:
Also, why there are not "Kubernetes Authors" while in other (even new files) are?

Reply:
Someone probably copied the header from a file where code from the kubernetes codebase was copied over. Such files contain the original copyright notice since it is apache licensed and  our copyright notice because of the new code. Files which are purley from us should not contain a the kubernetes copyright notice.